docs/Concepts/Privacy/Flexible-PrivacyGroups.md

```diff
@@ -60,4 +60,44 @@ group ID and passes the ID to Besu when creating a privacy group.
     participants. If you create a privacy group with an existing privacy group ID, the existing
     privacy group is overwritten.
 
-    To ensure unique privacy group IDs, using 256-bit SecureRandom is recommended.
+    To ensure unique privacy group IDs, we recommend using 256-bit SecureRandom.
+
+## Multi-tenancy
+
+When using [multi-tenancy](Multi-Tenancy.md) with flexible privacy groups, each user provides a JSON Web Token (JWT)
+which allows Besu to check that the user has access to functionality and data associated with a privacy group.
+
+Using multi-tenancy with flexible privacy groups is more complex than with [offchain privacy groups](Privacy-Groups.md)
+because users may try to access a privacy group they were once a member of but later removed from.
```

Comment:
because 
* when they are added to a group, the user gets access to all existing data in the privacy group
* even after being removed from a privacy group, users retain access to already existing data in the privacy group, up to the block where they were removed (the removal block)
* however a removed user does not have access to data in the privacy group that happens after they were removed